tools/toitlsp/lsp/compiler/path/path.go

```diff
@@ -21,6 +21,14 @@ import (
 	"strings"
 )
 
+const (
```

Comment:
This is currently duplicated from uri/convert.go, but Go doesn't allow cyclic imports, so removing the duplication would require a refactoring.
I might do it in a follow-up PR.